bitcoin/src/blockdata/script.rs

```diff
@@ -400,6 +400,16 @@ impl Script {
       Builder::new()
     }
 
+    /// Returns `Some(PublicKey)` if a public key is present
```

Comment:
> if a *valid* key is present

Also the documentation doesn't match the standard style of having a short summary. I suggest:

> Returns the public key if this script is P2PK with a valid public key.

Reply:
Thanks, documentation isn't exactly my strong suit so I appreciate the help